sw/airborne/subsystems/gps/gps_ubx.c

```diff
@@ -157,14 +172,50 @@ void gps_ubx_read_message(void)
         gps_ubx.state.svinfos[i].azim = UBX_NAV_SVINFO_Azim(gps_ubx.msg_buf, i);
       }
     } else if (gps_ubx.msg_id == UBX_NAV_STATUS_ID) {
+#if ! USE_GPS_UBX_RTCM
       gps_ubx.state.fix = UBX_NAV_STATUS_GPSfix(gps_ubx.msg_buf);
+#endif
       gps_ubx.status_flags = UBX_NAV_STATUS_Flags(gps_ubx.msg_buf);
       gps_ubx.sol_flags = UBX_NAV_SOL_Flags(gps_ubx.msg_buf);
+    } else if (gps_ubx.msg_id == UBX_NAV_RELPOSNED_ID) {
+      uint8_t version = UBX_NAV_RELPOSNED_VERSION(gps_ubx.msg_buf);
```

Comment:
Should be `#if USE_GPS_UBX_RTCM` here, else `struct gps_relposned` is undeclared when `USE_GPS_UBX_RTCM` is undefined